.github/workflows/tests-smoke.yml

```diff
@@ -0,0 +1,35 @@
+name: Test [smoke]
+
+on: pull_request
+
+concurrency:
+  group: '${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}'
+  cancel-in-progress: true
+
+jobs:
+  test-matrix:
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, windows-latest ]
+      fail-fast: false
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: 17
+          cache: 'maven'
+      - uses: gradle/gradle-build-action@v2
+        env:
+          ORG_GRADLE_PROJECT_org.jetbrains.dokka.javaToolchain.test: 11
+        with:
+          gradle-home-cache-cleanup: true
+      - name: Run tests under Ubuntu/Macos
+        if: matrix.os == 'ubuntu-latest' || matrix.os == 'macos-latest'
+        run: ./gradlew clean test --stacktrace
+      - name: Run tests under Windows
+        if: matrix.os == 'windows-latest'
+        # Running tests without additional flags on windows agents leads to some very strange
+        # JVM crashes for some reason. Most likely a problem of Gradle/GitHub/Windows server
+        run: ./gradlew clean test --stacktrace --no-daemon --parallel --max-workers=1
```

Comment:
Copy-pasted it from the old workflow code, not sure what the problem is or was exactly...